python/src/trezorlib/firmware/models.py

```diff
@@ -0,0 +1,170 @@
+    boardloader_sigs_needed: int
+    bootloader_keys: t.Sequence[bytes]
+    bootloader_sigs_needed: int
+    firmware_keys: t.Sequence[bytes]
+    firmware_sigs_needed: int
+
+
+TREZOR_ONE_V1V2 = ModelKeys(
+    production=True,
+    boardloader_keys=(),
+    boardloader_sigs_needed=-1,
+    bootloader_keys=(),
+    bootloader_sigs_needed=-1,
+    firmware_keys=[
+        bytes.fromhex(key)
+        for key in (
+            "04d571b7f148c5e4232c3814f777d8faeaf1a84216c78d569b71041ffc768a5b2d810fc3bb134dd026b57e65005275aedef43e155f48fc11a32ec790a93312bd58",
+            "0463279c0c0866e50c05c799d32bd6bab0188b6de06536d1109d2ed9ce76cb335c490e55aee10cc901215132e853097d5432eda06b792073bd7740c94ce4516cb1",
+            "0443aedbb6f7e71c563f8ed2ef64ec9981482519e7ef4f4aa98b27854e8c49126d4956d300ab45fdc34cd26bc8710de0a31dbdf6de7435fd0b492be70ac75fde58",
+            "04877c39fd7c62237e038235e9c075dab261630f78eeb8edb92487159fffedfdf6046c6f8b881fa407c4a4ce6c28de0b19c1f4e29f1fcbc5a58ffd1432a3e0938a",
+            "047384c51ae81add0a523adbb186c91b906ffb64c2c765802bf26dbd13bdf12c319e80c2213a136c8ee03d7874fd22b70d68e7dee469decfbbb510ee9a460cda45",
+        )
+    ],
+    firmware_sigs_needed=3,
+)
+
+TREZOR_ONE_V1V2_DEV = ModelKeys(
+    production=False,
+    boardloader_keys=(),
+    boardloader_sigs_needed=-1,
+    bootloader_keys=(),
+    bootloader_sigs_needed=-1,
+    firmware_keys=[
+        bytes.fromhex(key)
+        for key in (
+            "032c0b7cf95324a07d05398b240174dc0c2be444d96b159aa6c7f7b1e668680991",
+            "02edabbd16b41c8371b92ef2f04c1185b4f03b6dcd52ba9b78d9d7c89c8f221145",
+            "03665f660a5052be7a95546a02179058d93d3e08a779734914594346075bb0afd4",
+            "0366635d999417b65566866c65630d977a7ae723fe5f6c4cd17fa00f088ba184c1",
+            "03f36c7d0fb615ada43d7188580f15ebda22d6f6b9b1a92bff16c6937799dcbc66",
+        )
+    ],
+    firmware_sigs_needed=3,
+)
+
+TREZOR_ONE_V3 = ModelKeys(
+    production=True,
+    boardloader_keys=(),
+    boardloader_sigs_needed=-1,
+    bootloader_keys=(),
+    bootloader_sigs_needed=-1,
+    firmware_keys=[
+        bytes.fromhex(key)
+        for key in (
+            "032300c1bb4539fcbfca2590bda3dd2093826f4ae437bddecc1a2e72520764ff7a",
+            "0233baeaebc94a2a3e8b11f39a7133dbf427be292fcbceb887d71ef51e85395a19",
+            "0357091fa254b55233d0bb4c48e106c91b92fd0788ebed9d3a916719f44c76c015",
+        )
+    ],
+    firmware_sigs_needed=2,
+)
+
+TREZOR_ONE_V3_DEV = ModelKeys(
+    production=False,
+    boardloader_keys=(),
+    boardloader_sigs_needed=-1,
+    bootloader_keys=(),
+    bootloader_sigs_needed=-1,
+    firmware_keys=[
+        bytes.fromhex(key)
+        for key in (
+            "037308e14077161c365dea0f5c80aa6c5dba34719e825bd23ae5f7e7d2988adb0f",
+            "039c1b2460e343712e982e0732e7ed17f60de4c933065b7170d99c6e7fe7cc7f4b",
+            "03152b37fdf126111274c894c348dcc975b57c115ee24ceb19b5190ac7f7b65173",
+        )
+    ],
+    firmware_sigs_needed=2,
```

Comment:
the signing script in debug_signatures actually fill all three signatures, which causes the validation to fail. we should increase this to 3 or modify the script

Reply:
will be handled elsewhere